Lesson4/src/com/home/model/Person.java

```diff
@@ -18,14 +18,14 @@ public Person(int age, String name, String surname, int height, char gender) {
         this(age, name);
         this.height = height;
         this.gender = gender;
-        this.surname = surname;
+        this.surname = surname.toLowerCase().replaceFirst("^\\w", surname.substring(0, 1).toUpperCase());
```

Comment:
you also need to get rid of leading and ending spaces here and in surname

Reply:
you still need to trim this and name

Reply:
now it's looks like "this.surname = surname.trim().toLowerCase().replaceFirst("^\\w", surname.substring(0, 1).toUpperCase());"
IDK why it not changed((

Reply:
it's ok, i see it